plugin/external/pluginEntry.go

```diff
@@ -270,8 +277,8 @@ func (e *externalPluginEntry) schema() (*EntrySchema, error) {
 		// As a sanity check, ensure that the methods specified in the entry's schema
```

Comment:
I don't think this test is working right yet. Trying to start Wash with an external plugin, I got
```
{"Boltwash":{"children":["Group"],"description":null,"label":"bolt","metadata_schema":null,"methods":["list","schema"],"partial_metadata_schema":null,"signals":null,"singleton":true},"Group":{"children":["Target"],"description":null,"label":"group","metadata_schema":null,"methods":["list","schema"],"partial_metadata_schema":null,"signals":null,"singleton":null},"Target":{"description":null,"label":"target","metadata_schema":null,"methods":["list","exec","schema"],"partial_metadata_schema":null,"signals":null,"singleton":null}}
expected something like:
{
	"type_id_one":{
		"label": "one",
		"methods": ["list"],
		"children": ["type_id_two"]
	},
	"type_id_two":{
		"label":"two",
		"methods": ["read"]
	}
}
```

Reply:
Oh, this is happening because I defined `list` to return an empty list, so there are no child classes to declare for the schema yet. Works fine if I comment out the `list` implementation.

Reply:
Ah yeah, that makes sense.